### PR TITLE
Add documentation for calibration project and changelog entry

### PR DESCRIPTION
**Pull request by krlberry**
## Description
Adds documentation to each of the apps updated for #4303 mentioning that if running with a spiceinited cube, no local kernels are needed. Also adds one CHANGELOG entry for this project. 

## Related Issue
#4303

## Motivation and Context
#4303 documentation and wrapup

## How Has This Been Tested?
Documentation-only change.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ release.
 ## [Unreleased]
 
 ### Added
-
+- The following calibration applications were updated to not require local mission-specific SPICE kernels when working with spiceinited cubes: amicacal, ctxcal, lrowaccal, moccal, mdiscal, hical, hicalbeta, vikcal, and gllssical. This makes it possible to first run spiceinit using the spice server and then run these calibration applications without ever needing to download mission-specific kernels. [#4303](https://github.com/USGS-Astrogeology/ISIS3/issues/4303)
```
Comment (scsides) [inline CHANGELOG.md:39]: Probably should talk about this being a new feature, but the old behavior is still there. If spiceinit has not been run then the apps run as they used to and kernels area is still required

File: isis/src/hayabusa/apps/amicacal/amicacal.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@ xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Applica
   </brief>
 
   <description>
-  This program calibrates images from the Messenger mission's MDIS instrument.
+  This program calibrates images from the Hayabusa mission's AMICA instrument.
```
Comment (scsides) [inline isis/src/hayabusa/apps/amicacal/amicacal.xml:11]: wow, hard to believe we missed that one

**Review by scsides — CHANGES_REQUESTED** [review]
Odd how these all look similar

File: isis/src/hayabusa/apps/amicacal/amicacal.xml (hunk 2 above)
Comment (krlberry) [reply isis/src/hayabusa/apps/amicacal/amicacal.xml:11]: I was surprised, too!

**Review by scsides — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (krlberry) [reply CHANGELOG.md:39]: Good feedback, thanks! Updating...

**Pull request merged**